src/RecoveryServices/RecoveryServices.Backup.Autorest/README.md

```diff
@@ -0,0 +1,109 @@
+
+``` yaml
+# it's the same options as command line options, just drop the double-dash!
+branch: c94569d116a82ee11a94c5dfb190650dd675a1bf
+require:
+  - $(this-folder)/../readme.azure.noprofile.md
+input-file:
+  - $(repo)/specification/recoveryservicesbackup/resource-manager/Microsoft.RecoveryServices/stable/2023-02-01/bms.json
+title: RecoveryServices
+directive:
+  - where:
+      verb: New
+      subject: ProtectionPolicy
+      variant: ^CreateViaIdentityExpanded$|^CreateExpanded$|^CreateViaIdentity$
+    remove: true
+  - where:
+      verb: New
+      subject: ProtectionPolicy
+      variant: Create
+    hide: true
+  - where:
+      verb: Remove
+      subject: ProtectionPolicy
+      variant: DeleteViaIdentity
+    remove: true
+  - where:
+      verb: Get
+      subject: ProtectedItem
+      variant: GetViaIdentity
+    remove: true
+  - where:
+      verb: Get
+      subject: ProtectionPolicy
+      variant: GetViaIdentity
+    remove: true
+  - where:
+      subject: ProtectionPolicy
+    set:
+      subject: BackupPolicy
+  - where:      
+      subject: Job|BackupEngine|OperationResult|OperationStatuses|ProtectableItem|Item|ProtectionContainer|ProtectionIntent|EncryptionConfig|StorageConfigsNonCrr|VaultConfig|BackupStatus|BackupUsageSummary|JobDetail|OperationStatus|PrivateEndpointConnection|RecoveryPoint|RecommendedForMove|ResourceGuardProxy|SecurityPiN|ItemLevelRecoveryConnection|Restore|Cancellation|ValidateOperation|ResourceGuardProxyDelete|ProtectableContainer|Prepare|FeatureSupport
+    remove: true
+  - where:      
+      verb: Start
+    remove: true
+  - where:      
+      verb: Set
+      subject: BackupPolicy
+    remove: true
+  - no-inline:
+    - DailyRetentionSchedule
+    - HourlySchedule
+    - MonthlyRetentionSchedule
+    - Settings
+    - SubProtectionPolicyTieringPolicy
+    - TieringPolicy
+    - WeeklyRetentionSchedule
+    - YearlyRetentionSchedule
+  - from: source-file-csharp
+    where: $
+    transform: $ = $.replace('internal Microsoft.Azure.PowerShell.Cmdlets.RecoveryServices.Models.Api20230201.IProtectionPolicy Property', 'public Microsoft.Azure.PowerShell.Cmdlets.RecoveryServices.Models.Api20230201.IProtectionPolicy Property');
+  - from: source-file-csharp
+    where: $
+    transform: $ = $.replace('internal Microsoft.Azure.PowerShell.Cmdlets.RecoveryServices.Models.Api20230201.IRetentionPolicy RetentionPolicy', 'public Microsoft.Azure.PowerShell.Cmdlets.RecoveryServices.Models.Api20230201.IRetentionPolicy RetentionPolicy');
+  - from: source-file-csharp
+    where: $
+    transform: $ = $.replace('internal Microsoft.Azure.PowerShell.Cmdlets.RecoveryServices.Models.Api20230201.ISchedulePolicy SchedulePolicy', 'public Microsoft.Azure.PowerShell.Cmdlets.RecoveryServices.Models.Api20230201.ISchedulePolicy SchedulePolicy'); 
+```
+
+## Alternate settings
+
+This section is only activated if the `--make-it-rain` switch is added to the command line
+
+``` yaml $(make-it-rain)
```

Comment:
What is the purpose of this Alternate Settings section? This looks like not a standard attribute for the README.md. Is it possible to remove the whole section?

Reply:
yes we can remove it, if that's the standard. we referred Az.DataProtection

Reply:
> yes we can remove it, if that's the standard. we referred Az.DataProtection

--make-it-rain is a switch for autorest command. Generally in PowerShell, we don't need it. If you have no idea of its purpose, then please remove it.